packages/atomic/scripts/start-cdn.mjs

```diff
@@ -0,0 +1,223 @@
+import chalk from 'chalk';
+import {spawn} from 'child_process';
+import fs from 'fs/promises';
+import path from 'path';
+
+const currentDir = import.meta.dirname;
+const siteDir = path.resolve(currentDir, '../dev-cdn');
+
+const getVersionFromPackageJson = async (packageName, versionType) => {
+  const packageJsonPath = path.resolve(
+    currentDir,
+    `../../../packages/${packageName}/package.json`
+  );
```

Comment:
Bump the dev node req to gte 22.14.0 and use https://nodejs.org/docs/latest-v22.x/api/module.html#modulefindpackagejsonspecifier-base